### ISSUE TITLE
A11y: Explanatory text for the mandatory fields not placed in front of the form

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7058]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The explanation ‘The fields marked with an asterisk (*) are mandatory fields.’ is shown after the form.

![Image](https://github.com/user-attachments/assets/4f7d0193-31b9-4cbc-b243-ae7986a9a52d)

Blind people go through a page sequentially from ‘element to element’ or ‘from line to line’ and would not know what is meant by the asterisk when they see it for the first time. People who use magnification also only see a section of the screen and may not notice the information further down.

*_Expected behaviour:_*

The message should be placed in front of the form.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag:
- visit the registration page

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40822**



[created from NEXT-40822, comment 541489]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40822, comment 541555]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40822, comment 541556]

**Comment by Phil23** [comment]
Hey @73nici, 

The accessibility of forms was completely reworked with Shopware 6.6.10.0 so I recommend updating to the latest minor version. It will also get extended support, including accessibility changes that might be introduced later.

The message about required fields is just a visual requirement. The fields will have proper `aria-required` attributes, so the screen reader already notifies users if a field is required when they come across it.
